ext/webidl/00_webidl.js

```diff
@@ -844,8 +846,21 @@
           opts,
         );
       }
-      const keys = ReflectOwnKeys(V);
       const result = {};
+      // Fast path for common case (not a Proxy)
+      if (!core.isProxy(V)) {
+        for (const key in V) {
+          if (!ObjectPrototypeHasOwnProperty(V, key)) {
+            continue;
+          }
+          const typedKey = keyConverter(key, opts);
+          const value = V[key];
+          const typedValue = valueConverter(value, opts);
+          result[typedKey] = typedValue;
+        }
+      }
+      // Slow path if Proxy (e.g: in WPT tests)
```

Comment:
I might be missing something here, but just by looking at the code a `return` or `else` might be missing, since like this is runs both fast and slow path?

Reply:
That was smart ... Good catch! I accidentally messed this up in the tweak commit (https://github.com/denoland/deno/pull/12286/commits/c27e43a793131e8c8b1f1087893144e182f5ecfb)